cranelift/codegen/src/isa/riscv64/inst/vector.rs

```diff
@@ -234,6 +248,13 @@ impl fmt::Display for VecAluOpRRR {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
             VecAluOpRRR::Vadd => write!(f, "vadd.vv"),
+            VecAluOpRRR::Vsub => write!(f, "vsub.vv"),
+            VecAluOpRRR::Vmul => write!(f, "vmul.vv"),
+            VecAluOpRRR::Vmulh => write!(f, "vmulh.vv"),
+            VecAluOpRRR::Vmulhu => write!(f, "vmulhu.vv"),
+            VecAluOpRRR::Vand => write!(f, "vand.vv"),
+            VecAluOpRRR::Vor => write!(f, "vor.vv"),
+            VecAluOpRRR::Vxor => write!(f, "vxor.vv"),
```

Comment:
If these all follow the same pattern, one thing I found helpful for AVX was do do something like:

```rust
let mut s = format!("{self:?}");
s.make_ascii_lowercase();
s.push_str(".vv");
f.write_str(&s)
```

which can cut down on codegen times and additionally make this a bit easier to maintain. If the instructions have all sorts of different names though this may not work out well